File: Java/Projects/Calculator.java
import java.util.Scanner;

public class Calculator {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        double number1, number2, result;
        char operator;

        System.out.print("Input the first number : ");
        number1 = input.nextDouble();
        System.out.print("Input the operator (+, _, *, /)");
        operator = input.next().charAt(0);
        System.out.print("Input the second number : ");
        number2 = input.nextDouble();


        switch (operator) {
            case '+' -> {
                result = number1 + number2;
                System.out.println(number1 + " + " + number2 + " = " + result);
            }
            case '-' -> {
                result = number1 - number2;
                System.out.println(number1 + " - " + number2 + " = " + result);
            }
            case '/' -> {
                result = number1 / number2;
                System.out.println(number1 + " / " + number2 + " = " + result);
            }
            case '*' -> {
                result = number1 * number2;
                System.out.println(number1 + " * " + number2 + " = " + result);
        }
    }
}
}

File: Go/calculator.go
package main  
  
import "fmt"  
  
func main() {  
 var operator string  
 var number1, number2 int  
 fmt.Print("Please enter First number: ")  
 fmt.Scanln(&number1)  
 fmt.Print("Please enter Second number: ")  
 fmt.Scanln(&number2)  
 fmt.Print("Please enter Operator (+,-,/,%,*):")  
 fmt.Scanln(&operator)  
 output := 0  
 switch operator {  
 case "+":  
  output = number1 + number2  
  break  
 case "-":  
  output = number1 - number2  
 case "*":  
  output = number1 * number2  
 case "/":  
  output = number1 / number2  
 case "%":  
  output = number1 % number2  
 default:  
  fmt.Println("Invalid Operation")  
 }  
 fmt.Printf("%d %s %d = %d", number1, operator, number2, output)  
}  
